sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -230,6 +225,7 @@ export interface SendBatchOptions extends OperationOptions {
 
 // @public
 export interface SubscribeOptions extends TracingOptions {
+    fallbackPositions?: EventPosition | Map<string, EventPosition>;
```

Comment:
are we in sync with Python on the naming here?

Reply:
cc @YijunXieMS

Reply:
cc @chradek

Reply:
It looks like Python calls it [starting_position](https://github.com/Azure/azure-sdk-for-python/blob/25c0a046c3c486c83ec4fbc9fbbdf253e1091cf8/sdk/eventhub/azure-eventhub/azure/eventhub/_consumer_client.py#L221).

Reply:
The name `fallback` makes sense as it is the fall back value to use in the absence of a checkpoint.

But the concept of checkpoints might not still be a familiar one when people get started with the library and are exploring the apis.

On the other hand, starting_position can be misleading for someone familiar with checkpoints who might assume that this value is respected event if checkpoint exists

:(

Reply:
@bterlson says `startPosition`